ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -605,6 +660,15 @@ objects:
         trap 'rc=$?; if test "${rc}" -eq 0; then touch /tmp/setup-success; else touch /tmp/exit; fi; exit "${rc}"' EXIT
         trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
+        while true; do
```

Comment:
This loop should have a `/tmp/exit` safety valve like [this one][1], so we don't hang if the `lease` container fails.

[1]: https://github.com/openshift/release/blob/44dd3116723fe2b4ec7ea4a8317aa48a418fd7b6/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L128-L131